src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/AddMissingTokensCodeCleanupProvider.vb

```diff
@@ -82,6 +82,7 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
 
                 ' can't/don't try to transform member access to invocation
                 If TypeOf name.Parent Is MemberAccessExpressionSyntax OrElse
+                   TypeOf name.Parent Is TupleElementSyntax OrElse
```

Comment:
I don't think so. The `name` is a `NameSyntax`, which cannot be a direct child of a `TupleTypeSyntax`.